config/jobs/kubernetes/release/release-config.yaml

```diff
@@ -70,6 +70,13 @@ presubmits:
         command:
         - make
         - test
```

Comment:
I'm not entirely sure what this `make test` does, but seemed like 9G would be plenty

Reply:
not sure either but we can look at https://prow.k8s.io/?job=pull-release-test => recent green run https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/release/2959/pull-release-test/1635375671163228160 

looks like unit tests, this is a pretty generous allocation for that. we can also merge and iterate